shell/assets/styles/base/_variables-classic.scss

```diff
@@ -0,0 +1,85 @@
+.classic {
```

Comment:
are these all lift and shifted from `_light.scss`?

Reply:
Correct - I introduced this so that we can share classic colors between light and dark themes, we can't implicitly fallback to these variables without `_light.scss` as the default anymore.